Jenkinsfile

```diff
@@ -0,0 +1,187 @@
+def MACHINE = 'none'
+def machine = 'none'
+def HOME = 'none'
+
+pipeline {
+    agent { label 'built-in' }
+
+    options {
+        skipDefaultCheckout()
+    }
+
+    stages {
+
+        stage('Get Machine') {
+            agent { label 'built-in' }
+            steps {
+                script {
+                    MACHINE = 'none'
+                    for (label in pullRequest.labels) {
+                        echo "Label: ${label}"
+                        if ((label.matches("CI-Hera-Ready"))) {
+                            MACHINE = 'hera'
```

Comment:
May be this is me being nit-picky, but `MACHINE` is the name of the machine in lower-case (e.g. `hera` here) and `machine` is the name of the machine in upper-case set on line 29. 
Would it make sense to make their names in the cases they are as we are explicitly having different names for the different cases (upper-case and lower-case)?

Reply:
yes sounds solid, update.